packages/editor/src/client/index.tsx

```diff
@@ -113,7 +116,9 @@ const onDOMContentLoaded = async () => {
           <FacebookProvider sdkLanguage={'en_US'}>
             <InstagramProvider>
               <TwitterProvider>
-                <HotApp />
+                <ShepherdTour steps={steps} tourOptions={options}>
```

Comment:
I suggest you move the whole `<ShepherdTour>` to it's own component. This way you can also remove the imports above from the `index.tsx` which we try to keep as clean as possible.